fuel-core/src/schema/tx/types.rs

```diff
@@ -4,7 +4,9 @@ use super::receipt::Receipt;
 use crate::model::FuelBlockDb;
 use crate::schema::contract::Contract;
 use crate::schema::scalars::{AssetId, Bytes32, HexString, Salt, TransactionId, U64};
+use crate::schema::tx::Arc;
```

Comment:
this seems like a strange way to import Arc?

Reply:
My bad, I think I listened to the rustc suggestion without thinking that this wasn't a special type and just plain `Arc`